ci/log.py

```diff
@@ -44,19 +44,29 @@ class Bcolors:
     BLUE = '\033[34m'
 
 
-def configure_default_logging(stdout_level=None):
+def configure_default_logging(
+    stdout_level=None,
+    force=False,
```

Comment:
is it not rather the case we do by default want the force?

Reply:
True.